### DISCUSSION TITLE
菜鸟求问 有没有完整新手教程

### DISCUSSION THREAD

**Discussion by xmsz** [openkraken/kraken#99] [Q&A] [answered]
flutter环境装了2个晚上 主要卡在各种网络问题 最后用镜像和代理解决

然后今天终于在真机运行起来了 🎉

但是后面不知道该怎么做了

我现在的需求是
- 做内部人员（几个人 都是ios）的工具，所以就是纯内部测试使用环境
- 需要动态化，因为要经常修改和调整，不想每次更新都打包

除了后面不知道怎么做，现在还有几个细节问题

除了后面不知道怎么做，现在还有几个细节问题
- app顶部url输入栏 真机输入后 直接白屏 但是模拟器可以 这是怎么回事
- 真机重新打开直接闪退 再也打不开了

**Comment by andycall** [comment, marked as answer]
文档还在写，预计很快就可以发布出来了

**Reply by xmsz** [reply]
Cool~

**Comment by andycall** [comment]
ios 闪退是证书问题，多人使用估计得申请个企业证书才行。个人开发者证书不行

**Reply by xmsz** [reply]
ok 我再研究研究

**Comment by wssgcg1213** [comment]
iOS 真机在不连 XCode 的情况下闪退是 Flutter 自身问题, 这个得看官方修复的进度~

**Reply by wssgcg1213** [reply]
正式包没有这个现象

**Reply by xmsz** [reply]
ok
